recipe/build_mamba.sh

```diff
@@ -0,0 +1,49 @@
+if [[ $PKG_NAME == "mamba" ]]; then
+	cd mamba
+	$PYTHON -m pip install . --no-deps -vv
+	if [ -d "$PREFIX/condabin" ]; then
+		echo "Adding link to mamba into condabin";
+		ln -s $PREFIX/bin/mamba $PREFIX/condabin/mamba
+	fi
+	exit 0
+fi 
+
+rm -rf build
+mkdir build
+cd build
+
+if [[ $PKG_NAME == "libmamba" ]]; then
+	cmake .. ${CMAKE_ARGS} 				\
+		-GNinja 						\
+		-DCMAKE_INSTALL_PREFIX=$PREFIX 	\
+		-DCMAKE_PREFIX_PATH=$PREFIX     \
+		-DBUILD_LIBMAMBA=ON 			\
+		-DBUILD_SHARED=ON               \
+		-DBUILD_MAMBA_PACKAGE=ON
+elif [[ $PKG_NAME == "libmambapy" ]]; then
+	# TODO finds wrong python interpreter!!!!
+	cmake .. ${CMAKE_ARGS} 				\
+		-GNinja 						\
+		-DCMAKE_PREFIX_PATH=$PREFIX     \
+		-DCMAKE_INSTALL_PREFIX=$PREFIX 	\
+		-DPython_EXECUTABLE=$PYTHON     \
+		-DBUILD_LIBMAMBAPY=ON
+fi
+
+ninja
+
+if [[ $PKG_NAME == "libmamba" ]]; then
+    strip libmamba/libmamba.so
+	strip mamba_package/mamba-package
```

Comment:
I would like to strip `mamba-package` executable but it looks like it doesn't work as expected

Reply:
we should also have `strip libmamba/libmamba.so*`